beacon-chain/db/kv/genesis_test.go

```diff
@@ -28,41 +29,70 @@ func testGenesisDataSaved(t *testing.T, db iface.Database) {
 	ctx := context.Background()
 
 	gb, err := db.GenesisBlock(ctx)
-	assert.NoError(t, err)
-	assert.NotNil(t, gb)
+	require.NoError(t, err)
+	require.NotNil(t, gb)
```

Comment:
I changed these to `require` after spending some time chasing a panic that would not occur if tests halted at a `NoError` assertion. `require` causing a fast failure makes these tests easier to debug.